mod/aalborg_theme/views/default/css/elements/misc.php

```diff
@@ -62,3 +62,7 @@
 	display: none;
 	padding: 0;
 }
+
+.elgg-highlight {
```

Comment:
How about .elgg-state-highlight and then use that with another class to make the effect happen.

For example:

```
.elgg-comment.elgg-state-highlight {
  background: #whatevs
}
```

This is consistent with how we use other `elgg-state-*` classes.

Or a custom `elgg-comment-highlight` class for comments.